pkg/testutils/lint/lint_test.go

```diff
@@ -1865,12 +1865,16 @@ func TestLint(t *testing.T) {
 			pkgs, err := packages.Load(
 				&packages.Config{
 					Mode: packages.NeedImports | packages.NeedName,
+					Dir:  crdbDir,
 				},
 				pkgPath,
 			)
 			if err != nil {
 				return errors.Wrapf(err, "error loading package %s", pkgPath)
 			}
+			if len(pkgs) == 0 {
```

Comment:
Small Nit: A comment explaining why this is check is required might help future readers. But definitely already much better than failing silently.